src/Provider/SecurityServiceProvider.php

```diff
@@ -57,8 +59,6 @@ function ($app) {
                 ];
             }
         );
-
-        $app->register(new SilexSecurityServiceProvider());
     }
```

Comment:
Yeah … badly done humour on my part … it should have read "Ooops… I forgot" :laughing: